src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -4,21 +4,38 @@ import * as educationProgram from '../definitions/educationProgram';
 import { uiSchema as uiSchemaDate } from '../../common/schemaform/definitions/date';
 import { civilianBenefitsLabel } from '../utils/helpers';
 
-const defaults = {
-  fields: [
-    'educationProgram',
-    'educationObjective',
-    'nonVaAssistance'
-  ],
-  required: [
-    // 'educationType',
-    // 'name'
-  ]
+
+// Exported like this so we can share the fields between the formConfig and the tests.
+export const schoolSelectionOptionsFor = {
```

Comment:
I just liked the idea of having these here so we can use them in our tests. Not that we _really_ need them in there, I don't think, but...It's super convenient.